docs/index.md

```diff
@@ -90,6 +90,48 @@ testImplementation('org.testcontainers:mysql') //no version specified
     These libraries in particular seem to be especially prone to conflicts with test code/application under test code. 
     As such, **these libraries are 'shaded' into the core testcontainers JAR** and relocated under `org.testcontainers.shaded` to prevent class conflicts.
 
+## Sponsors
+
+Testcontainers is on Github Sponsors! Please consider sponsoring us to sustain the project and help us improve it further.
+
+<iframe src="https://github.com/sponsors/testcontainers/button" title="Sponsor Testcontainers" height="35" width="107" style="border: 0;"></iframe>
+
+A huge thank you to our sponsors:
+
+### Bronze sponsors
+
+<div style="text-align:center; max-width: 128px; display: inline-block; margin: 5px;">
+    <a href="https://cirruslabs.org/">
+        <img src="sponsor_logos/cirrus_labs.jpg" style="width: 100%"/>
+        <p>Cirrus Labs</p>
+        <!-- via fkorotkov's sponsorship -->
+    </a>
+</div>
```

Comment:
@fkorotkov we're updating the sponsors section of the page, adding logos for bronze sponsors. Would you be happy for your generous $100/month sponsorship to be attributed to Cirrus Labs? Or would you prefer to keep it listed as a personal sponsorship?

Reply:
Thank you, @rnorth! Let's attributed it to the company because Test Containers is the crucial part of the integration tests for Cirrus CI. Do you mind to change the URL and the name to Cirrus CI? It's the main product and Test Containers are actually used for it and can be used with it.